scripts/run-integration-template.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+INTEGRATION=$1
+shift
+
+NANGO_SECRET_KEY_DEV=$(grep NANGO_SECRET_KEY_DEV .env | cut -d '=' -f2)
+
+cd integration-templates/$INTEGRATION
+
+echo $NANGO_SECRET_KEY_DEV
```

Comment:
I will not echo the secret key so it doesn't end up in our bash history

Reply:
Whoops. That was debug left over. Removing